pkg/util/sysctl/sysctl_linux.go

```diff
@@ -0,0 +1,46 @@
+//+build linux
+
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sysctl
```

Comment:
any reason to have this in `pkg/util/sysctl` and not `pkg/agent/util/systcl`?

Reply:
I'd recommend moving it honestly. I think it belongs with platform-specific code in the agent. It's very similar to `pkg/agent/util/ethtool`, and I would expect to find them in the same place.

Reply:
Done.